torch/csrc/jit/script/init.cpp

```diff
@@ -843,6 +843,16 @@ void initJitScriptBindings(PyObject* module) {
         TORCH_INTERNAL_ASSERT(name.name() == def.name().name());
         return script_compile_function(name, def, defaults, std::move(rcb));
       });
+  m.def("_try_get_compiled_func", [](const std::string& qualname) {
```

Comment:
I think this basically implements `CompilationUnit::find_function`, so let's move it to the pybind for `CompilationUnit` instead of making a free function. The current implementation pybound to `find_function` is actually `get_function`, so we'll need to rename that too.